### DISCUSSION TITLE
（好像是我电脑的问题？）https://travellings.link/第一次能跳转，第二次不能跳转

### DISCUSSION THREAD

**Discussion by StarWEB890** [travellings-link/travellings#1205] [寻求帮助] [answered]
这个问题我之前在Q群反映过一次，大致问题：
https://travellings.link/第一次能正常访问，也能正常跳转
但到第二次：
![BhwTZTVSGL](https://cdn.staticaly.com/gh/StarWEB890/TuChuang@master/images/BhwTZTVSGL.2ew7u99loo5c.webp)
（这个问题之前我没看见有人反映过，难道是我电脑的问题？）

**Comment by linlinzzo** [comment, marked as answer]
已在 #1218 中解决。
